config/v1/types_apiserver.go

```diff
@@ -184,7 +184,7 @@ type APIServerEncryption struct {
 	Type EncryptionType `json:"type,omitempty"`
 }
 
-// +kubebuilder:validation:Enum="";identity;aescbc
+// +kubebuilder:validation:Enum="";identity;aescbc;aesgcm
```

Comment:
use a technique for techpreview only https://github.com/openshift/api/blob/master/example/v1/types_stable.go#L72-L73 and get insta-merge.

You'll be able to remove the techpreview category when the testing works